cpp/csp/python/PyStructFastList_impl.h

```diff
@@ -511,6 +511,27 @@ static PyObject * PyStructFastList_Reversed( PyStructFastList<StorageT> * self,
     CSP_RETURN_NULL;
 }
 
+template<typename StorageT>
+static PyObject * PyStructFastList_reduce( PyStructFastList<StorageT> * self, PyObject * Py_UNUSED( ignored) )
+{
+    CSP_BEGIN_METHOD;
+    
+    typename VectorWrapper<StorageT>::IndexType sz = self -> vector.size();
+    PyObject * args = PyTuple_New( sz );
+    PyObjectPtr list = PyObjectPtr::own( toPython( self -> vector.getVector(), self -> arrayType ) );
+    for( typename VectorWrapper<StorageT>::IndexType index = 0; index < sz; ++index )
+    {
+        PyObject * value = PyList_GET_ITEM( list.ptr(), index );
+        Py_INCREF( value );
+        PyTuple_SET_ITEM( args, index, value );
+    }
+    PyObject * result = Py_BuildValue( "O(O)", &PyList_Type, args );
```

Comment:
I think you might need an incref on PyList_Type here, as per the docs Py_BuildValue ")" steals a reference:
https://docs.python.org/3/c-api/arg.html#other-objects

Reply:
Actually, a new strong reference is created for Py_BuildValue(), it's not created for Parse..() functions. So I believe there's no need for incref.